### PR TITLE
Added check for space for heading tags.

### PR DESCRIPTION
**Pull request by bvelasquez**
The #hashtag should not render as a H1 Header, but just normal typeface. The "# hastag" should render as a header on its own line. Added a space check in the regex.

### CODE REVIEW

**Comment by bvelasquez** [comment]
Do you merge any pull requests anymore?

**Comment by Benjamin-Dobell** [comment]
Thanks for reporting this.

I'd like to fix this, unfortunately this pull request currently consists of 183 changes, where as it should be just a single character change i.e. replacing the `*` in the regex with `+`.

If you'd like to update your pull request accordingly, then I'll happily merge it. Otherwise, it's easy enough for me to make the change; just let me know.

File: MarkdownView.js [hunk 1]
```diff
@@ -59,7 +59,7 @@ const DefaultRules : Rules = Object.freeze(mergeRules(
   ),
   {
     heading: {
-      match: SimpleMarkdown.blockRegex(/^ *(#{1,6}) *([^\n]+?) *#* *(?:\n *)*\n/),
+      match: SimpleMarkdown.blockRegex(/^ *(#\s{1,6}) *([^\n]+?) *#* *(?:\n *)*\n/),
```
Comment (Benjamin-Dobell) [inline MarkdownView.js]: The added `\s` seems to be incorrect. That will match `# header`, `#  header`, `#   header`, `#    header`, `#     header` and `#      header`.

You'll just want to replace the second asterisks (`*`) in the regex with a `+`, as presently we're looking for zero or more spaces after the 1-6 hashes (`#`), where as we should be looking for one or more spaces.

Comment (bvelasquez) [reply MarkdownView.js]:
```
match: SimpleMarkdown.blockRegex(
          /^ *(#{1,6})\s *([^\n]+?) *#* *(?:\n *)*\n/
        )
```
The last push I did. Do I need to make a new pull request?

Comment (bvelasquez) [reply MarkdownView.js]: Oh, I see what happened. Prettier changed the file. ugh.

Comment (Benjamin-Dobell) [reply MarkdownView.js]: No need to submit a new pull request.

```
git rebase -i HEAD~2
```

then drop the first commit (`d`) and edit the second one (`e`) to correct the regex.

Once you're done editing:

```
git rebase --continue
git push -f origin master
```

**Comment by bvelasquez** [comment]
Still messing with the formatting, even after following your instruction. No newline for some reason.

**Comment by Benjamin-Dobell** [comment]
Manually merged https://github.com/Benjamin-Dobell/react-native-markdown-view/commit/c04986537e9faae5ef685eb254b7fd3e0b8d780b 😄

**Comment by bvelasquez** [comment]
Thank you! I've disabled prettier for any future git PR's. What a mess that
was.

Barry

**Pull request closed without merging**